### ISSUE TITLE
Feature discovery doesn't work as expected in card.

### ISSUE THREAD

**Issue by mevansLA** [Dogfalo/materialize#5654]
https://codepen.io/anon/pen/EQvvXx

When a feature discovery / tap target block is placed on a card, the position or location of the block when displayed is incorrect. eg:
https://i.gyazo.com/8f512feeb361083b4ac48827bae7aeda.png

The issue is resolved by moving the feature discovery block to the body of the page.  Perhaps this should be documented?

**Comment by acburst** [comment]
it seems like you fixed the issue? You are correct in that the safest way for it to work is to put it in the root level of the DOM

**Comment by mevansLA** [comment]
Yep, I was thinking it might be helpful to add this to the documentation.

**Comment by mevansLA** [comment]
Just noticed that this is also present on the materializecss website when viewing 1.0.0-beta but not when viewing 0.100.2
http://materializecss.com/feature-discovery.html

**Comment by r0b78** [comment]
today the version 1.0.0 dont work well in card , any solution?

**Comment by frikishaan** [comment]
Can anyone please tell me does it also affect if my `<body>` has some **margin** and **padding**?
